src/sentry/static/sentry/app/components/group/releaseStats.jsx

```diff
@@ -106,15 +67,21 @@ const GroupReleaseStats = createReactClass({
     );
   },
 
+  getEnvironment(envName) {
+    let defaultEnv = EnvironmentStore.getDefault();
```

Comment:
`getDefault` looks like it can be null so we should protect against this.

Reply:
Not necessary for this PR but we probably want to move away from querying the store like this. We want the flow of data from store -> component. (This will also help prepare us for redux migration).

So when location.query.environment changes, we'd fire an actionCreator which will update the store. And in this component we would just use Reflux.connect.

Reply:
Yeah, definitely agree. I might try to tackle this at a later stage.